internal/docstore/docstore.go

```diff
@@ -31,7 +31,7 @@ import (
 //
 // A Document can be represented as a map[string]int or a pointer to a struct. For
 // structs, the exported fields are the document fields.
-type Document = interface{}
+type Document interface{}
```

Comment:
Just curious, why did you make this change?

Reply:
I wasn't sure why it was necessary to do an alias vs a type definition? Is that a better practice?

Reply:
If `Document` is a separate type, then anyone calling `Get`, `Put`, etc. will have to cast their struct or map to it. 

I made it an alias so that it basically serves as documentation, that `Get` etc. take Go types that are valid documents. 

We can discuss this further, but in any case it shouldn't be part of this PR.

Reply:
Reverted.